src/components/SplitHorizontal/SplitHorizontal.less

```diff
@@ -0,0 +1,37 @@
+.lucid-SplitHorizontal {
+	overflow: hidden;
+
+	& > &-inner {
+		height: 100%;
+		display: flex;
+		flex-direction: column;
+
+		& > .lucid-SplitHorizontal-TopPane,& > .lucid-SplitHorizontal-BottomPane {
+			overflow: auto;
+			flex: 1;
+
+			&.lucid-SplitHorizontal-is-secondary {
+				flex: 0;
+			}
+		}
+
+		& > .lucid-SplitHorizontal-Divider {
+			height: 6px;
+			background: @color-gray;
+			box-sizing: border-box;
+			cursor: ns-resize;
+		}
+	}
+
+	// fixes flicker in chrome for both the animated and non-animated collapse/expand
+	& > .lucid-SplitHorizontal-inner {
+
+		& > .lucid-SplitHorizontal-TopPane {
+			.no-safari-flicker();
```

Comment:
yes i can, i separated them intentionally because these are for crossbrowser compat bug fixes, and the above are for component styling, so they are in a section on their own...they compile down to the same css, but this organization was intentional. I can merge the two sections tho if it looks better.